css/styles.css

```diff
@@ -50,6 +50,100 @@ h3 {
   margin: 20px 5px;
 }
 
+.placeholder-container {
+  display: flex;
+  justify-content: center;
+  font-size: 1.4rem;
+  margin-top: 30px;
+}
+
+.new-entry-btn-container {
+  display: flex;
+  justify-content: center;
+}
+
+.new-entry-btn {
+  width: 100px;
+  height: 40px;
+  background-color: #95c7d4;
+  border: #85b2be;
+  border-radius: 3px;
+  box-shadow: 0 0 2px 1px #678d97;
+  color: #fff;
+  font-size: 1.1rem;
+}
+
+.form-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 450px;
+  margin-top: 30px;
+}
+
+form {
+  display: flex;
+  justify-content: center;
+  width: 350px;
+  height: 450px;
```

Comment:
Are both of these height properties sizing the same element? It might be worth doing a review to see what if any styling is duplicated or no longer necessary in your CSS 🤔

Reply:
The heights do different things, but since they share a few properties, I just put the common ones in its own class.